operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -932,7 +953,25 @@ func (r *Cluster) FullImageName() string {
 	return fmt.Sprintf("%s:%s", r.Spec.Image, r.Spec.Version)
 }
 
-// ExternalListener returns external listener if found in configuration. Returns
+// AllKafkaAPIExternalListeners returns all the Kafka external listeners.
+func (r *Cluster) AllKafkaAPIExternalListeners() []KafkaAPI {
```

Comment:
NIT: In my opinion `ExternalListener` function should be renamed to `FirstExternalListener`. Then `AllKafkaAPIExternalListeners` could become `KafkaExternalListeners`. The Prefix `All` seems obsolete.

Reply:
> Is there a plan to have new version of CRD so we can deprecate it?

Yes, but it take a while